### DISCUSSION TITLE
Using on Windows?

### DISCUSSION THREAD

**Discussion by aydgn** [deepjyoti30/ytmdl#135] [Q&A]
How to use this on Windows 10?. I have Python 3.9.1 installed.

**Comment by deepjyoti30** [comment]
Hey @aydgn You can check [this](https://github.com/deepjyoti30/ytmdl/issues/51#issuecomment-737221619) comment on a detailed instruction on how to use on Windows.

After following all the above steps, you **might** face an error. As far as I know, the latest version of Python might be installed as `py` in the console which means in the `cmd` you'll need to call `py` instead of `python` to execute python.

So you can try running `ytmdl` directly from `cmd` and if it fails then try doing `py ytmdl`, it should work!
